src/main/java/gov/usgs/cida/pubs/transform/CrossrefTransformer.java

```diff
@@ -130,49 +130,85 @@ public void write(Object result) throws IOException {
 	/**
 	 * 
 	 * @param pub
-	 * @return true if result was written successfully, false otherwise.
 	 * @throws IOException 
 	 */
-	protected boolean writeResult(Publication<?> pub) throws IOException {
-		boolean success = false;
+	protected void writeResult(Publication<?> pub) throws IOException {
```

Comment:
The only time getMinimalRecordComment would need to be updated is to give
the user a specific message, otherwise it should be handled with a generic message.
will do the two steps.